### DISCUSSION TITLE
Design Prompt: How is this better than LogSeq and other BASB or Digital Garden?

### DISCUSSION THREAD

**Discussion by BradKML** [reorproject/reor#322] [Ideas]
Reor can be seen as a kind of document manager and chat agent (kinda like [Khoj](https://github.com/khoj-ai/khoj)) but it can surely do more as a note taker and writer. This would make it more closely resemble LogSeq but without LLM support natively (needs [plugins](https://github.com/omagdy7/ollama-logseq) to work). Some questions to consider:
1. Would hyperlinks to other notes (and adding aliases for notes) be a viable way to enhance writing experience in Reor?
2. Can Reor archive texts from the internet, extract text, and derive summaries or extra thoughts? (ditto for PDF/Zotero)
3. How far is Reor from turning into a creative engine that can synthesize ideas and generate new forms of thought?

**Comment by brandonmarkgaia-hub** [comment]
I’m exploring a parallel approach to AI + PKM by treating GitHub itself as the “garden substrate” and layering a codex / lore system on top of it.

It’s less of a traditional note app and more of a public, machine-readable myth / knowledge garden with books, echoes (issues) and status JSON for agents. I’d love to compare notes on how Reor thinks about becoming a “creative engine that can synthesize ideas and generate new forms of thought.”

Mine is tried and tested
